test/unit/replace_version_test.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+from exasol.toolbox.tools import replace_version
+
+
+def test_retrieve_issue_templates():
```

Comment:
try to make a parametrized test out of this.
https://docs.pytest.org/en/7.1.x/how-to/parametrize.html